datalakehouse-api/Dockerfile

```
@@ -0,0 +1,51 @@
+
+# ── Application code ──────────────────────────
+COPY *.py .
+
+# ── Warehouse mount-point ─────────────────────
+# Mount your local warehouse here:
+#   docker run -v /opt/Tazama_Warehouse:/opt/Tazama_Warehouse ...
+RUN mkdir -p /opt/Tazama_Warehouse
+
+VOLUME ["/opt/Tazama_Warehouse"]
+
+EXPOSE 8181 ${DATALAKEHOUSE_API_PORT:-8282}
```

Hadolint (2.14.0)

[error] unexpected '}'
expecting '/', a new line followed by the next instruction, or digit

(DL1000)